src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/ClassDefHandler.java

```diff
@@ -79,6 +79,10 @@ public void checkIndentation() {
                     logError(ident, "ident", lineStart);
                 }
             }
+            if (getMainAst().getType() == TokenTypes.RECORD_DEF) {
```

Comment:
Block checks the parens too. Why is this better in class handler and not block handler?

Reply:
It wasn't, thanks for questioning this.  Must have made some false assumption along the way while working on this.  All code in this comment , https://github.com/checkstyle/checkstyle/pull/9157#discussion_r575545157 and https://github.com/checkstyle/checkstyle/pull/9157#discussion_r575545238 has been removed.